ru.linux_gaming.PortProton.metainfo.xml

```diff
@@ -0,0 +1,144 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+	<id>ru.linux_gaming.PortProton</id>
+
+	<name>PortProton</name>
+	<summary>Software for playing Microsoft Windows games and launchers</summary>
+	<summary xml:lang="ru">Программа для запуска игр и лаунчеров с Microsoft Windows</summary>
+
+	<developer id="linux-gaming.ru">
+		<name>Mikhail Tergoev</name>
+	</developer>
+
+	<url type="homepage">linux-gaming.ru</url>
+	<url type="bugtracker">https://github.com/Castro-Fidel/PortWINE/issues</url>
+	<url type="donation">https://linux-gaming.ru/t/podderzhat-proekt/123/</url>
+	<url type="vcs-browser">https://github.com/Castro-Fidel/PortWINE</url>
+	<url type="help">https://linux-gaming.ru/c/portproton/4</url>
+
+	<metadata_license>CC0-1.0</metadata_license>
+	<project_license>MIT</project_license>
+
+	<description>
+		<p>
+			PortProton is a project designed to make it easy and convenient to run Windows games on Linux for both beginners and advanced users. The project strives to make running games (and other software) as easy as possible, but at the same time provides flexible settings for advanced users
+		</p>
+		<p xml:lang="ru">
+			PortProton – это проект, разработанный для того, чтобы сделать легким и удобным запуск Windows игр на Linux как для начинающих, так и для продвинутых пользователей. Проект стремится сделать запуск игр (и другого программного обеспечения) максимально простым, но в то же время предоставляет гибкие настройки для продвинутых пользователей
+		</p>
+	</description>
+
+	<launchable type="desktop-id">ru.linux_gaming.PortProton.desktop</launchable>
+
+	<screenshots>
+		<screenshot type="default">
+			<caption>Installed</caption>
+			<image>https://raw.githubusercontent.com/Boria138/PortProton_flatpak/main/screenshots/installed.png</image>
+		</screenshot>
+		<screenshot>
+			<caption>Auto Instals</caption>
+			<image>https://raw.githubusercontent.com/Boria138/PortProton_flatpak/main/screenshots/auto_install.png</image>
```

Comment:
Screenshots should be from a tag or a commit

Reply:
> Screenshots should be from a tag or a commit

Ah, why can't you take a direct link from github ?

Reply:
Same as above.

Reply:
> Same as above.

I asked the creator to create a tag with metainfo icons and stuff, but now I can't figure out how I can unpack the release into manifest properly https://github.com/Castro-Fidel/PortProton_ALT/releases/tag/v1.5.1.

Reply:
I left the portproton script in flathub because in upstream only the one for the native version is available

Reply:
> but now I can't figure out how I can unpack the release into manifest properly

The way you added it seems fine, what isn't working?

Reply:
> > but now I can't figure out how I can unpack the release into manifest properly
> 
> The way you added it seems fine, what isn't working?

When I asked I wasn't sure how to do it right, now I did and checked that everything works